client/src/components/layouts/guide.css

```diff
@@ -170,3 +170,7 @@
 .content {
   outline: 0;
 }
+
+.title span:nth-of-type(2) {
```

Comment:
Why so precisely? Look at line 117 in this file and try to understand why this is not working?

Reply:
instead of ```.title span:nth-of-type(2) ``` would just setting the color property  of  the ```.title``` selector be sufficient?

Reply:
I'm going over this.

Reply:
Alright, I was under the impression that the caret too will get affected by the change but it seems fine so I've done the change